app-editors/okteta/okteta-0.26.13.ebuild

```diff
@@ -45,7 +45,7 @@ DEPEND="
 	>=kde-frameworks/kio-${KFMIN}:5
 	>=kde-frameworks/kjobwidgets-${KFMIN}:5
 	>=kde-frameworks/knewstuff-${KFMIN}:5
-	>=kde-frameworks/kparts-${KFMIN}:5
+	kparts? ( >=kde-frameworks/kparts-${KFMIN}:5 )
```

Comment:
This should go after `crypt?` to keep sorting, unless asturm disagrees (I don't recall if kde@ has different style guidelines here).